### PR TITLE
Refactoring: add missing imports

### PR DESCRIPTION
**Pull request by afrokick**
- add missing imports from `meteor/XXX`
- remove some unused imports

### CODE REVIEW

File: lib/docsize_cache.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 let LRU = Npm.require('lru-cache');
-let crypto = Npm.require('crypto');
```
Comment (afrokick) [inline lib/docsize_cache.js:2]: doesn't use

File: lib/hijack/http.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ if (Package['fetch']) {
       const response = originalCall.apply(this, arguments);
 
       response
-        .then((res) => {
```
Comment (afrokick) [inline lib/hijack/http.js:66]: unused

File: lib/hijack/wrap_routers.js [hunk 3]
```diff
@@ -1,5 +1,3 @@
-import Fibers from 'fibers';
```
Comment (afrokick) [inline lib/hijack/wrap_routers.js:1]: unused

File: lib/hijack/wrap_server.js [hunk 4]
```diff
@@ -1,5 +1,3 @@
-let Fiber = Npm.require('fibers');
```
Comment (afrokick) [inline lib/hijack/wrap_server.js:1]: unused

File: lib/utils.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ export function haveAsyncCallback (args) {
   return typeof lastArg === 'function';
 }
 
-UniqueId = function (start) {
```
Comment (afrokick) [inline lib/utils.js:9]: unused

**Review by zodern — APPROVED** [review]

**Pull request merged**